tt_stl/CMakeLists.txt

```
@@ -0,0 +1,35 @@
+add_library(tt_stl INTERFACE)
+add_library(TT:STL ALIAS tt_stl)
+
+target_sources(
+    tt_stl
+    PUBLIC
+        FILE_SET tt_stl
+        TYPE HEADERS
+        BASE_DIRS tt_stl
```

```suggestion
        BASE_DIRS .
```

Assuming we need to package these up (all signs point to yes 🤪 ), then they'll preserve anything deeper than `BASE_DIR`.  So assuming we want them to stay under a `tt_stl` dir, then we want the BASE_DIR above that.  A little confusing here b/c we have `tt_stl/tt_stl`.  In an installed environment we'd want only one (eg: `/usr/include/tt_stl/foo.h`.

---

we can fix this later :-)